Tests/SonarQube.Common.UnitTests/Infrastructure/OutputCaptureScope.cs

```diff
@@ -15,6 +15,9 @@ namespace SonarQube.Common.UnitTests
     /// Utility class to capture the standard console output and error streams.
     /// Disposing the class resets the console to use the standard streams.
     /// </summary>
+    /// <remarks>Note: attempts to set Console.ForegroundColor and Console.BackgroundColor
+    /// don't work if the console output is being redirected, so we have no easy way of
+    /// capturing the colour in which the text is written.</remarks>
```

Comment:
Unfortunately, this makes the console output rather hard to test other than manually. I'll add some tests for the new scope class.